### PR TITLE
Bugfix: isSack() function revision... AGAIN.

### PR DESCRIPTION
**Pull request by RayDeeUx**
so yeah. hypixel admins did a big brain moment where this ONE button in the composter menu gets treated as a false positive under my initial revision of `isSack()` so i revised the function ONE LAST TIME... at least, i hope.

anyways have fun

### CODE REVIEW

**Review by hannibal002 — CHANGES_REQUESTED** [review]
At this point, you can just do isSack(stack)

**Comment by RayDeeUx** [comment]
> At this point, you can just do isSack(stack)

countergagument: i dont want to deal with kotlin's whole "tHiS vAriAbLe iS mUtAbLE" BS

File: src/main/java/at/hannibal2/skyhanni/features/misc/PocketSackInASackDisplay.kt [hunk 1]
```diff
@@ -4,6 +4,7 @@ import at.hannibal2.skyhanni.SkyHanniMod
 import at.hannibal2.skyhanni.events.GuiRenderItemEvent
 import at.hannibal2.skyhanni.events.LorenzToolTipEvent
 import at.hannibal2.skyhanni.utils.ItemUtils
+import at.hannibal2.skyhanni.utils.ItemUtils.cleanName
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/features/misc/PocketSackInASackDisplay.kt]: Please manually check your imports before commiting (or use IntelliJ auto orgianize imports on commit feature)

**Review by hannibal002 — CHANGES_REQUESTED** [review]
Please use IntelliJ already :D

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**